### PR TITLE
This should work now

### PR DESCRIPTION
**Pull request by jo-tud**
If you merge my master branch, it should work. Maybe I messed sth up with the branches.

### CODE REVIEW

**Comment by mNantern** [comment]
Yep it's better !

I just copy/paste here my previous remark (so everything is in one place):

Just one minor remark : can you put the "Due" item under "Uncategorized" ?

And in that PR, I saw all tasks marked as done in the "overdue" item (they were not shown in the previous PR).
Is it possible to change the order under "Due" ?

* Overdue
* Today
* This week
 * This month

**Comment by jo-tud** [comment]
Regarding changing the order ... I haven't found a simple solution, yet. Only alphabetical sorting is easy. Maybe you have an idea?

**Comment by jo-tud** [comment]
I realized the correct order.

File: qtodotxt/lib/filters.py [hunk 1]
```diff
@@ -103,6 +107,121 @@ def isMatch(self, task):
...
+class DueOverdueFilter(BaseFilter):
+    """
+    Task list filter allowing only incomplete tasks that are overdue.
+
+    """
+    def __init__(self, dueRange):
+        BaseFilter.__init__(self, dueRange)
+
+    def isMatch(self, task):
+        if (not task.due) and (task.is_complete):
```
Comment (mNantern) [inline qtodotxt/lib/filters.py]: should be "or" here instead of "and"

File: qtodotxt/ui/views/filters_tree_view.py [hunk 2]
```diff
@@ -104,6 +120,7 @@ def _addDefaultTreeItems(self, tree):
             FilterTreeWidgetItem(None, ['Complete'], CompleteTasksFilter(), getIcon('x.png'))
         tree.addTopLevelItems([
             self._incompleteTasksItem,
+            self._dueItem,
```
Comment (mNantern) [inline qtodotxt/ui/views/filters_tree_view.py]: That should be one line under (after uncategorizedTasks...)

Comment (jo-tud) [inline qtodotxt/ui/views/filters_tree_view.py]: I don't really agree. In my opinion *Due* is related to time, just as *Pending*. *Uncategorized* is related to *Context* and *Projects* and should be placed before or after (above *Complete*).

Comment (mNantern) [inline qtodotxt/ui/views/filters_tree_view.py]: I understand your point. You are right !

**Pull request merged**